src/lib/stitching/exchange/__tests__/stitching.test.ts

```diff
@@ -344,4 +344,116 @@ describe("resolving a stitched conversation", () => {
       data: { commerceOrder: { isInquiryOrder: false } },
     })
   })
+
+  it("resolves conversation field on CommerceOfferOrder", async () => {
+    const allMergedSchemas = await incrementalMergeSchemas(schema, 2)
+    const query = gql`
+      {
+        commerceOrder(id: 4200) {
+          conversation {
+            items {
+              item {
+                ... on Artwork {
+                  title
+                }
+              }
+            }
+          }
+        }
+      }
+    `
+
+    // Mock the resolvers for just an OfferOrder with a conversation id.
+    // The part we are testing is the step that goes from a order
+    // to the conversation.
+    addMockFunctionsToSchema({
+      preserveResolvers: true,
+      schema: allMergedSchemas,
+      mocks: {
+        Query: () => ({
+          commerceOrder: (_root, _params) => {
+            return {
+              __typename: "CommerceOfferOrder",
+              impulseConversationId: "conversation-id",
+            }
+          },
+        }),
+      },
+    })
+
+    const result = await graphql(
+      allMergedSchemas,
+      query,
+      {},
+      {
+        conversationLoader: jest.fn(() =>
+          Promise.resolve({
+            items: [
+              {
+                item_type: "Artwork",
+                properties: {
```

Comment:
After many tries i finally got this loader to test the full graphql request handling stack.